filesystems/filesystems.tex

```diff
@@ -738,25 +757,25 @@ \subsection{The `sticky' bit}
 
 When a directory's sticky bit is set only the file's owner, the directory's owner, and the root user can rename or delete the file.
 This is useful when multiple users have write access to a common directory.
-A common use of the sticky bit is for the shared and writable \keyword{/tmp} directory.
+A common use of the sticky bit is for the shared and writable \keyword{/tmp} directory where many user's files may be stored, but users should not be able to access files belonging to other users.
 
 To set the sticky bit, use \keyword{chmod +t}.
 
 \begin{verbatim}
-aneesh$ mkdir sticky
-aneesh$ chmod +t sticky
-aneesh$ ls -l
-drwxr-xr-x  7 aneesh aneesh    4096 Nov  1 14:19 .
-drwxr-xr-x 53 aneesh aneesh    4096 Nov  1 14:19 ..
-drwxr-xr-t  2 aneesh aneesh    4096 Nov  1 14:19 sticky
-aneesh$ su newuser
-newuser$ rm -rf sticky
-rm: cannot remove 'sticky': Permission denied
-newuser$ exit
-aneesh$ rm -rf sticky
-aneesh$ ls -l
-drwxr-xr-x  7 aneesh aneesh    4096 Nov  1 14:19 .
-drwxr-xr-x 53 aneesh aneesh    4096 Nov  1 14:19 ..
+    aneesh$ mkdir sticky
+    aneesh$ chmod +t sticky
+    aneesh$ ls -l
+    drwxr-xr-x  7 aneesh aneesh    4096 Nov  1 14:19 .
+    drwxr-xr-x 53 aneesh aneesh    4096 Nov  1 14:19 ..
+    drwxr-xr-t  2 aneesh aneesh    4096 Nov  1 14:19 sticky
+    aneesh$ su newuser
+    newuser$ rm -rf sticky
+    rm: cannot remove 'sticky': Permission denied
+    newuser$ exit
+    aneesh$ rm -rf sticky
+    aneesh$ ls -l
+    drwxr-xr-x  7 aneesh aneesh    4096 Nov  1 14:19 .
+    drwxr-xr-x 53 aneesh aneesh    4096 Nov  1 14:19 ..
```

Comment:
Let's keep the spaces out of verbatim too.

This should also be an lstlisting.